src/server/static/js/lines.js

```diff
@@ -224,10 +224,50 @@ var layout = {
   }
 };
 
+const DOWNLOAD_PLOT_SCALE = 1;
+let getDownloadPlotTitle = () => {
+  let regions = baseData.regions;
+
+  if(!(selected.region in regions)){
+    return "COVID-19 Forecast";
+  }
+
+  return `COVID-19 Forecast for ${regions[selected.region].name}`;
+}
 var plotlyConfig = {
   displaylogo: false,
   responsive: true,
-  scrollZoom: false
+  scrollZoom: false,
+  modeBarButtonsToAdd: [{
+    name: 'Download plot',
+    icon: Plotly.Icons.camera,
+    click: (gd) => saveImage(gd, {name: selected.region, scale: DOWNLOAD_PLOT_SCALE, format: "png", background: "black", compose: ($canvas, plot, width, height) => {
```

Comment:
When referencing things defined on `window`, please reference it explicitly (`window.saveImage`)

Reply:
But its not a global right? I forget how these things work. It would be really nice to set up that trans-compilation.

Reply:
Changed.